__exekutir__/inventory/host_vars/kommandir.yml

```diff
@@ -81,10 +81,13 @@ git_cache_args:
     - repo: "https://github.com/autotest/autotest.git"
       recursive: False
       cachepath: 'autotest'
+      depth: 2
     - repo: "https://github.com/autotest/autotest-docker.git"
       recursive: True
       cachepath: 'autotest-docker'
+      depth: 2
     - repo: "https://github.com/python-bugzilla/python-bugzilla.git"
       recursive: False
       cachepath: "python-bugzilla"
+      # Need to fetch everything to find specific tagged version
```

Comment:
I realize that this comment is supposed to be interpreted as "this is why you're not seeing a `depth` parameter here", but a future reviewer will think that the comment applies to the `version` line... and that makes no sense. 

If my one-line-conditional suggestion below works out, then you can just remove this comment and life is good.

If my one-line-conditional suggestion does not work, and we have to live with this code, then this comment should be updated to say something like "Do not include a depth parameter here! [...this is why...]"